Rubberduck.VBEditor.VB6/SafeComWrappers/VB/Window.cs

```diff
@@ -20,6 +21,33 @@ public IntPtr Handle()
             return (IntPtr)HWnd;
         }
 
+        private IntPtr _unknown;
+        public IntPtr IUnknown
+        {
+            get
+            {
+                if (IsWrappingNullReference)
+                {
+                    return IntPtr.Zero;
+                }
+
+                if (_unknown == IntPtr.Zero)
+                {
+                    try
+                    {
+                        _unknown = Marshal.GetIUnknownForObject(Target);
+                        Marshal.Release(_unknown);
+                    }
+                    catch
+                    {
+                        // If GetIUnknownForObject threw us here, we're fine. If Marshal.Release threw us here, we're probably screwed
+                        // anyway, so we might as well just wait for the zombie process when the VBE tries to close.
```

Comment:
Same comment RE: `Debug.Warn` - it might be useful to know to show in the log that we got thrown here because of `Marhsal.Release` vs. `Marshal.GetIUnknownForObject` since the latter isn't necessarily a problem... *in theory*.

Reply:
Hmmm... I'd take suggestions on the wording for that. AFAICT there isn't a way to delineate what went wrong there. Should I nest the 2 calls in the `try` and `catch` each one individually?

Reply:
if we can ignore the exception for one of the use-cases, then we should separate that from a possibly fatal error

Reply:
@Vogel612  - it shouldn't be "fatal" in the sense that it will bring the process down, but I agree - I'll nest the `try`s.